cli/tests/integration_tests.rs

```diff
@@ -5082,6 +5082,8 @@ fn concat_bundle(
     bundle_line_count += 1;
   }
 
+  bundle.push_str("function importScripts() {}\n");
```

Comment:
```suggestion
  bundle.push_str("function importScripts() {}\n");
  bundle_line_count += 1;
```

source mapping wont be correct otherwise.

Reply:
The linter is warning me that `bundle_line_count` is not read after that line.
I'm putting the function declaration at the very top of the bundle then.